Dockerfile

```diff
@@ -29,8 +29,7 @@ WORKDIR protocol
 RUN npm install
 RUN scripts/buildContracts.sh
 
-# Command to run Voting system. The setup above could probably be extracted to a base Docker image, but that may require
-# modifying the directory structure more.
+# Command to run any script in the UMA repository. To run the voting
```

Comment:
Maybe delete the part of this comment after the period?

Reply:
Oops, my bad. Will do.

Reply:
Forgot to re-save after I made changes to the comments.